### PR TITLE
small text size fix for TM-2999

### PR DESCRIPTION
**Pull request by pveland621**
ACs:
1. Verify 'exit client view' on client search view banner is same text size as 'client dashboard'

### CODE REVIEW

File: src/sass/_clientView.scss [hunk 1]
```diff
@@ -34,15 +34,15 @@
     > div:nth-child(2) {
       display: flex;
       flex: 2;
-      font-size: 1.6rem;
+      font-size: 1.8rem;
```
Comment (elizabeth-jimenez) [inline src/sass/_clientView.scss:37]: let's stay away from relative units

File: src/sass/_clientView.scss [hunk 2]
```diff
@@ -28,21 +28,21 @@
 
     > div:nth-child(1) {
       flex: 1;
-      font-size: 1.6rem;
```
Comment (elizabeth-jimenez) [inline src/sass/_clientView.scss:31]: is it just 3 divs?

Comment (pveland621) [reply src/sass/_clientView.scss:31]: looks that way, yeah
image

Comment (elizabeth-jimenez) [reply src/sass/_clientView.scss:31]: it is in the code, i mean if there are other divs under `usa-banner-inner` in the ui - you'll want to be aware of inadvertently modifying and other ones with my suggestion to consolidate

Comment (pveland621) [reply src/sass/_clientView.scss:31]: looks like it's just being used in gov and state banner, and beta header. gov and state banner are only referenced in test files and i dont think beta header will be affected

Comment (pveland621) [reply src/sass/_clientView.scss:31]: also i think the 2nd div child had a slightly bigger font size, not sure if can consolidate font-size in that case

Comment (elizabeth-jimenez) [reply src/sass/_clientView.scss:31]: Yea, you're right - i didn't notice that before, sorry

**Review by elizabeth-jimenez — APPROVED** [review]

**Pull request merged**